app/view_models/auction_list_item.rb

```diff
@@ -43,92 +43,130 @@ def eligibility_label
     end
   end
 
-  def available?
-    auction_status.available?
+  def github_repo
+    auction.github_repo
   end
 
-  def bid_button_partial
-    if current_user_can_bid?
-      'auctions/bid_button'
+  def project_label_partial
+    if auction.github_repo
+      'auctions/project_label'
     else
       'components/null'
     end
   end
 
-  def item_footer_partial
-    if available?
-      'auctions/bids_and_form'
-    else
-      'auctions/winning_bid'
-    end
+  def project_label
+    auction.github_repo.sub('https://github.com/', '')
   end
 
   def winning_bid_partial
+    if over? && !auction.bids.any?
+      'auctions/no_bids'
+    elsif auction.type == 'single_bid'
+      winning_single_bid_partial
```

Comment:
I reverted it :shrug: It seems like a delicate balance. Oh the narrow lines we walk!